extensions/resteasy-jaxb/deployment/src/test/java/io/quarkus/resteasy/jaxb/deployment/ProducesXMLTestCase.java

```diff
@@ -1,11 +1,18 @@
 package io.quarkus.resteasy.jaxb.deployment;
 
+import static org.hamcrest.Matchers.*;
```

Comment:
I missed that one but we usually avoid star imports.

Reply:
Will do.